Snippets/Snippets_3/DataBus/CleanupStrategy/Usage.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Snippets3.DataBus.CleanupStrategy
+{
+    using System;
+    using System.IO;
+    using Snippets3.DataBus.DataBusProperty;
+
+    public class Usage
+    {
+        #region FileLocationForDatabusFiles
+        public void Handle(MessageWithLargePayload message)
+        {
+            string filename = Path.Combine(@"c:\databus_files\", message.LargeBlob.Key);
```

Comment:
Discussed this @SimonCropp 
Also why do we need the Handle? What are we illustrating in a message handler? 
Shouldn't the cleanup policy be part of a timer, as opposed to a message handler?

Removing files as part of the handler is not a good idea because file deletes aren't part of DTC and can transactions can rollback.